.gitignore

```diff
@@ -93,4 +93,5 @@ terraform.tfstate.backup
 .pemkey
 *.xml
 logs/
-*.log
\ No newline at end of file
+*.log
+*.json
```

Comment:
Good Idea we shouldn't add jsons to our repo

Reply:
Per OpenAI senpai ,"XML (Extensible Markup Language) and JSON (JavaScript Object Notation) are two distinct data formats used for structuring and exchanging information, but they serve similar purposes" . Agreed, no need to push entire datasets to the repo.